Automattic/wp-calypso
# Activity Log: New selector getActivityLog with dummy data populated hourly events

This adds a placeholder selector for getActivityLog until we're actually pulling data from the tree.  For testing, we're generating hourly events manually, based on the real start date from the `getRewindStartDate` selector.  

This is a bit of a hack, and is not intended to live like this forever.  It is only meant to get some "real" hourly events in there in order to test restores.  

To Test: 
- Make sure your site has a premium plan
- Check `state.activityLog.rewindStatus`: if your site ID has a `firstBackupDate: ` set, then you should see hourly events in the UI.
- If no backupdate is in the state, you will not see any events.
